bugbug/tools/code_review.py

```diff
@@ -369,8 +369,50 @@ def get_all_inline_comments(
                 yield diff_id, comments
 
 
+class SwarmReviewData(ReviewData):
+    def __init__(self):
+        self.auth = {
+            "user": get_secret("SWARM_USER"),
+            "password": get_secret("SWARM_PASS"),
+            "port": get_secret("SWARM_PORT"),
+            "instance": get_secret("INSTANCE"),
+        }
+
+    # return ReviewRequest object with patch_id = revision_id
```

Comment:
We could add this as docstrings instead of comments. Example: https://github.com/mozilla/bugbug/blob/3fd6ab0717391c3ae99bec83383fdd55ba520d7c/bugbug/bugzilla.py#L305-L314

Reply:
I think the full docstring is not needed here since this implements an interface. The documentation should be in the abstract class instead (I will take care of that later).

Thus, it may be more suitable to use a regular comment inside the method to describe why the patch ID is used as a revision ID.